features/bevragen/persoon/verblijfplaats/dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,268 @@
+# language: nl
+
+Functionaliteit: vertalen van gevraagde samengestelde of afgeleide verblijfplaats velden naar leveren van bron velden
+  Wanneer met fields wordt gevraagd om een veld dat wordt samengesteld uit verschillende andere velden in de GbaPersoon of als een veld dat moet worden meegeleverd wordt samengesteld uit verschillende andere velden in de GbaPersoon, dan worden deze benodigde velden geleverd.
+
+  Rule: wanneer een veld van verblijfplaats m.b.v. de alias 'verblijfplaatsBinnenland' wordt gevraagd worden alleen de velden die horen bij een binnenlandse verblijfplaats geleverd die nodig zijn voor het bepalen van het type binnenlands verblijfplaats.
+    - dit betreft alle velden uit groepen 11 (adres) en 12 (locatie)
+	- dit is het geval als de consumer niet geautoriseerd is voor buitenlandse adressen
```

Comment:
Is dit relevant? Een consumer die is geautoriseerd voor buitenlandse adressen, zou ook de alias kunnen gebruiken